ui/components/app/custom-spending-cap/custom-spending-cap.js

```diff
@@ -146,25 +182,43 @@ export default function CustomSpendingCap({
         >
           <label
             htmlFor={
-              value > (currentTokenBalance || error)
+              conversionGreaterThan(
```

Comment:
The same call appears three times in this file:
```
conversionGreaterThan(
                { value: Number(value), fromNumericBase: 'dec' },
                { value: Number(currentTokenBalance), fromNumericBase: 'dec' },
              )
```
Could you store that in a variable, and then use the variable throughout?
```
const valueIsGreaterThanBalance = conversionGreaterThan(
                { value: Number(value), fromNumericBase: 'dec' },
                { value: Number(currentTokenBalance), fromNumericBase: 'dec' },
              )
              ```